### ISSUE TITLE
Links suggestions: suggestions list doesn't scroll when using a keyboard

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#10820]
**Describe the bug**

When there are many suggested links (more than 10) and using a keyboard to highlight the suggested links, the list doesn't scroll.

**To Reproduce**
- create a post, add a block, select some text and start inserting a link
- using only the keyboard type something in the search field so you have more than 10 results
- only ~10 results are visible in the list (note: there are some CSS issues on current master)
- in my case, see screenshot below, I have 20 results and only 10 are initially visible:

<img width="711" alt="screen shot 2018-10-20 at 00 32 52" src="https://user-images.githubusercontent.com/1682452/47246892-99c96000-d400-11e8-99ce-54856e0297ed.png">

- now, use the keyboard down or up arrow keys to highlight the results in the list
- the "highlighting" moves through the list
- but when on the 11th result, the list doesn't scroll
- keep pressing arrow down and apparently nothing happens (while actually it is highlighting the results that are not visible)
- at the 21st key press, finally the first result on top is highlighted again

This way, it's impossible to actually _see_ the complete list of results.

Note that while focus should stay in the search field, the list should scroll when pressing down / up arrows. In the Classic Editor this works like a charme.

**Comment by afercia** [comment]
Just tested previous Gutenberg versions and this worked correctly in 2.9.0. Marking as regression.
